reducers/chatsReducer.js

```diff
@@ -65,8 +83,8 @@ function chatsReducer(state = chatsInitialState, action) {
             }
 
         case types.REMOVE_USER_FROM_CHAT:
-            const updatedChatWithoutUser = { 
-                ...Object.assign({}, state.currentChat), 
+            const updatedChatWithoutUser = {
+                ...Object.assign({}, state.currentChat),
```

Comment:
minor thing, this is redundent, ... is the same thing as Object.Assign, so you can just do 

{ ...state.currentChat, users: state.currentChat.users.filter(user => user.id !== payload.users_id)}

Reply:
Oooh, interesting. I didn't know the spread operator created a copy. Thanks!